snowshu/core/models/relation.py

```diff
@@ -108,25 +103,11 @@ def relation(self) -> str:
     def relation(self, value: str) -> None:
         self.name = value
 
-    @staticmethod
-    def quoted(val: str) -> str:
-        """Returns quoted value if appropriate."""
-        return f'"{val}"' if any({val.isupper(),
-                                  val.islower(), }) and ' ' in val else val
-
     def scoped_cte(self, string: Optional[str] = None) -> str:
         """ returns a CTE name scoped to the relation.
             If _string_ is provided, this will be suffixed to the name."""
         return "__".join([self.database, self.schema, self.name, string])
 
-    def typed_columns(self, data_type_mappings: dict) -> str:
```

Comment:
I don't recognize this function, but I'm assuming this function isn't used anywhere since we're removing it.

Reply:
Yes. It's not being used anywhere